help/destinations/catalog/crm/outreach.md

```diff
@@ -0,0 +1,219 @@
+
+#### Gather Outreach credentials {#gather-credentials}
+
+Note down the items below before you authenticate to the [!DNL Outreach] CRM destination:
+
+| Credential | Description |
+|---|---|
+| Email | Your [!DNL Outreach] account email |
+| Password | Your [!DNL Outreach] account password |
+
+#### Setup custom field labels
+
+[!DNL Outreach] supports custom fields for [prospects](https://support.outreach.io/hc/en-us/articles/360001557554-Outreach-Prospect-Profile-Overview). Refer to [How To add a custom field in Outreach](https://support.outreach.io/hc/en-us/articles/219124908-How-To-Add-a-Custom-Field-in-Outreach) for additional guidance. For ease of identification it is recommended to manually update the labels to their corresponding segment names instead of keeping the defaults. For example as below:
+
+[!DNL Outreach] settings page for prospects displaying custom fields.
+![[!DNL Outreach] custom fields defaults](../../assets/catalog/crm/outreach/outreach-custom-fields.png)
+
+[!DNL Outreach] settings page for prospects displaying custom fields with *user-friendly* labels matching to the segment names. You can view the segment status on the prospect page against these labels.
+![[!DNL Outreach] custom fields with labels](../../assets/catalog/crm/outreach/outreach-custom-field-labels.png)
+
+> [!NOTE]
+>
+> Label names are for ease of identification only, they are not used when updating prospects.
+
+## Supported identities {#supported-identities}
+
+[!DNL Outreach] supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+| `OutreachId` | <ul><li>[!DNL Outreach] identifier, this is a numeric value corresponding to the prospect profile.</li><li>The ID must match the ID within the [!DNL Outreach] URL for the prospect being updated.</li><li>Refer to the [[!DNL Outreach] documentation](https://api.outreach.io/api/v2/docs#update-an-existing-resource) for further details.</li></ul>|Mandatory |
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+---------|----------|---------|
+| Export type | **[!UICONTROL Profile-based]** | <ul><li>You are exporting all members of a segment, together with the desired schema fields *(for example: email address, phone number, last name)*, according to your field mapping.</li><li> Platform segment statuses are exported to [!DNL Outreach] by specifying their corresponding custom field attribute in [!DNL Outreach], within the **[!UICONTROL Activate Destination]** > **[!UICONTROL Schedule segment export]** > **[!UICONTROL Mapping ID]** field.</li></ul> |
```

Comment:
On line 74 we need to rephrase the second bullet point:

- Platform segment statuses are exported to [!DNL Outreach] by entering their [!DNL Outreach] custom field attribute in the [!UICONTROL Mapping ID] field, during the [!UICONTROL Schedule segment export] step.

Reply:
There is a section #schedule-segment-export-example should I also link to it ?

Reply:
Yes, please link to that section, but also rephrase the bullet point.